src/EntityFrameworkCore.Projectables/Services/ProjectionExpressionResolver.cs

```diff
@@ -15,11 +15,14 @@ public LambdaExpression FindGeneratedExpression(MemberInfo projectableMemberInfo
 
             var expression = GetExpressionFromGeneratedType(projectableMemberInfo);
```

Comment:
There should be no need to call this if `projectableAttribute.UseMemberBody` is not null

Reply:
Comment accepted